osbrain/agent.py

```diff
@@ -754,15 +781,51 @@ def _process_sub_event(self, socket, serialized):
     def str2bytes(self, message):
         return message.encode('ascii')
 
+    def _is_address_internal(self, address):
```

Comment:
I am not sure this is necessary (i.e.: if we know the `AgentAddress` we should already know about the serialization method).